pages/blog/index.page.tsx

```diff
@@ -85,7 +85,7 @@ export default function StaticMarkdownPage({ blogPosts }: { blogPosts: any[] })
               <div className='bg-blue-100 hover:bg-blue-200 cursor-pointer font-semibold text-blue-800 inline-block px-3 py-1 rounded-full mb-4 text-sm'>
                 {recentBlog[0].frontmatter.type}
               </div>
-              <div className='text-h1mobile md:text-h1 font-semibold'>
+              <div className='text-h1mobile md:text-h1 font-semibol text-stroke-1'>
                 {recentBlog[0].frontmatter.title}
               </div>
```

Comment:
```suggestion
              <h1 className='text-h1mobile md:text-h1 font-semibol text-stroke-1'>
                {recentBlog[0].frontmatter.title}
              </h1>
```

Reply:
Thanks for the suggestion Ben. I added and looks great.